content/docs/rendering-elements.md

```diff
@@ -1,75 +1,77 @@
 ---
 id: rendering-elements
-title: Rendering Elements
+title: Rendering av element
 permalink: docs/rendering-elements.html
 redirect_from:
   - "docs/displaying-data.html"
 prev: introducing-jsx.html
 next: components-and-props.html
 ---
 
-Elements are the smallest building blocks of React apps.
+Element är de minsta byggstenarna i React-applikationer.
 
-An element describes what you want to see on the screen:
+Ett element beskriver vad som skall visas på skärmen:
 
 ```js
 const element = <h1>Hello, world</h1>;
```

Comment:
Since we started to translate text inside code I feel we should keep it consistent throughout.

Reply:
Fixing. 👍